src/Update.hs

```diff
@@ -286,7 +288,7 @@ updateAttrPath log mergeBase updateEnv@UpdateEnv {..} attrPath = do
           mbLastCommitMsg <- lift $ Git.findAutoUpdateBranchMessage packageName
           tell $ Alt mbLastCommitMsg
           unless hasUpdateScript do
-            lift $ checkExistingUpdate log updateEnv mbLastCommitMsg attrPath
+            lift $ checkExistingUpdate log updateEnv mbLastCommitMsg attrPath oldSrcUrl
```

Comment:
This doesn't look right to me—checking for open PRs that target the old src seems useless at best.

I figure we want to do this new check after the rewrites have run in both the `hasUpdateScript` and `not hasUpdateScript` cases, so maybe it should be pulled out of `checkExistingUpdate` and put in its own function.